content/en/docs/concepts/services-networking/service.md

```diff
@@ -117,7 +117,8 @@ spec:
       targetPort: 9376
 ```
 
-Applying this manifest creates a new Service named "my-service", which
+Applying this manifest creates a new Service named "my-service" with the default
+ ClusterIP [service type](#publishing-services-service-types). The Service 
```

Comment:
nit: whitespace

Reply:
@shannonxtreme,
Thank you for pointing it out; I've removed the whitespace.